pkg/appmanager/as3Manager.go

```diff
@@ -91,8 +91,8 @@ func (appMgr *Manager) getAS3ObjectFromTemplate(
 					continue
 				}
 
-				// filter out empty json objects and pool objects
-				if cl := getClass(v); cl == "" || cl == "Pool" {
+				// filter out pool objects
+				if cl := getClass(v); cl == "Pool" {
```

Comment:
Could you make this a separate commit with a proper commit mentioning what problem it addresses? It will be easy for us to look back in the future.